site/src/sections/Navbar/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import pattern from "../../assets/images/brand/SVG/service-mesh-patterns-side.svg";
+import { Link } from "gatsby";
+import Layer5 from "../../assets/images/brand/SVG/Layer5.svg";
+import { Navigation, Toggle , Navbox , Hamburger , LogoWrap , NavItem } from "./navbar.style";
+
+
+const Navbar1 = () => {
+  const [navbarOpen, setNavbarOpen] = useState(false);
+
+  return (
+    <Navigation>
+      <LogoWrap as={Link} to="/">
+        <img className="pattern" src={pattern} alt="pattern" />
+      </LogoWrap>
+      <Toggle
+        navbarOpen={navbarOpen}
+        onClick={() => setNavbarOpen(!navbarOpen)}
+      >
+        {navbarOpen ? <Hamburger open /> : <Hamburger />}
+      </Toggle>
+      {navbarOpen ? (
+        <Navbox>
+          <NavItem to="/">Book Chapters</NavItem>
+          <NavItem to="/">Meet the Authors</NavItem>
+          <NavItem to="/">Patterns in action</NavItem>
+          <NavItem to="/about">About <img className="layer5" src={Layer5} alt="Layer5 logo" /></NavItem>
+          <NavItem to="/get"><button><span className="label">Get it Now</span></button></NavItem>
+        </Navbox>
+      ) : (
+        <Navbox open>
+          <NavItem to="/">Book Chapters</NavItem>
+          <NavItem to="/">Meet the Authors</NavItem>
+          <NavItem to="/">Patterns in action</NavItem>
+          <NavItem to="/about">About <img className="layer5" src={Layer5} alt="Layer5 logo" /></NavItem>
+          <NavItem to="/get"><button><span className="label">Get it Now</span></button></NavItem>
+        </Navbox>
+      )}
```

Comment:
I don't think you need to duplicate the whole code for the mobile menu, just using a class when `navbarOpen` is true, would do. The same applies to the `Hamburger`, if the `navbarOpen` is false just return ''